chore(components-angular): migrate to Angular 22 (#7928)
swisspost/design-system: .changeset/major-coats-lose.md

---
'@swisspost/design-system-components-angular': major
---

> Update the library to be compatible with Angular 22. The peer dependencies were updated and v10 cannot be used with lower Angular versions.

Reviewer: ```suggestion
Updated the library to be compatible with Angular 22. The peer dependencies were updated and v10 can no longer be used with lower Angular versions.
```
